dune

```diff
@@ -13,14 +13,14 @@
 
 (library
  (name curl_lwt)
- (public_name curl_lwt)
+ (public_name curl-lwt)
```

Comment:
Is it right that the archive name is `curl_lwt.cma`? I think it is best to keep the number of names to a minimum, so `curl_lwt` for the Findlib and OPAM packages seems right. What good be a reason to introduce a different name for the library and the Findlib/OPAM packages?

Reply:
Right. My bad. Let's not change that.